lbrynet/daemon/Daemon.py

```diff
@@ -236,9 +236,8 @@ def setup(self):
         self.exchange_rate_manager.start()
 
         yield self._initial_setup()
-        yield threads.deferToThread(self._setup_data_directory)
-        migrated = yield self._check_db_migration()
-        yield self.storage.setup()
+        yield self.component_manager.setup()
+        self.storage = self.component_manager.get_component("database").storage
```

Comment:
@hackrush01 you can remove this since you've removed the references to Daemon.storage (to use Daemon.session.storage)

Reply:
`self.storage` is still needed [here](https://github.com/lbryio/lbry/pull/1164/commits/168c0eebd7e34e63ed8f48e887c4a96e1af7b1c0#diff-309f9696e2149ec1146e8674e77e4e49R497)

Reply:
Removed in the latest commit